website/api/controllers/deliver-apple-csr.js

```diff
@@ -4,24 +4,31 @@ module.exports = {
   friendlyName: 'Deliver Apple CSR',
 
 
-  description: 'Generate and deliver a signed certificate signing request to a requesting user\'s email address.',
+  description: 'Generate and optionally deliver a signed certificate signing request to a requesting user\'s email address.',
 
-  extendedDescription: 'Uses the mdm-gen-cert binary to generate a signed CSR for the user and sends the result to the requesting user\'s email address',
+  extendedDescription: 'Uses the mdm-gen-cert binary to generate a signed CSR for the user and optionally sends the result to the requesting user\'s email address',
 
 
   inputs: {
     unsignedCsrData: {
       required: true,
       type: 'string',
       description: 'Base64 encoded CSR submitted from the Fleet server or `fleetctl` on behalf of the user.'
+    },
+    alt: {
+      required: false,
+      type: 'string',
+      description: 'Allows to customize the response type. If specified and set to "json" it returns the signed CSR contents in the response.'
```

Comment:
Can you change the name of this input and set a default value that defaults to the old behavior e.g.,
```suggestion
    delvieryMethod: {
      type: 'string',
      defaultsTo: 'email',
      isIn: ['email', 'json'],
      description: 'How the signed CSR will be delivered to the user. '
```

Reply:
TIL, thank you